commands/build.go

```diff
@@ -172,16 +173,6 @@ func (o *buildOptions) toControllerOptions() (*controllerapi.BuildOptions, error
 		}
 	}
 
-	annotations, err := buildflags.ParseAnnotations(o.annotations)
-	if err != nil {
-		return nil, err
-	}
-	for _, e := range opts.Exports {
-		for k, v := range annotations {
-			e.Attrs[k.String()] = v
-		}
-	}
-
```

Comment:
So here `--push` hasn't been translated to an export entry. Can you just link me the line where this happens for learning purposes?